apps/wing/src/commands/pack.ts

```diff
@@ -143,6 +146,11 @@ export async function pack(options: PackageOptions = {}): Promise<string> {
     // add "wing" top-level field
     pkgJson.wing = true;
 
+    if (!pkgJson.peerDependencies) {
+      pkgJson.peerDependencies = {};
+    }
+    pkgJson.peerDependencies["@winglang/sdk"] = "*";
+
```

Comment:
Forgot I left this in, need to double check if this is necessary. I ran into issues but I was also foolishly testing with pnpm...

Reply:
So I took this out, but I had to add the same thing to ts4w at least.

This is going to be a big issue I think: winglibs have a version of wingsdk that they expect, the cli has a version of wingsdk, and now the typescript framework also needs to be linked to a version.

For now I'll keep the problem isolated to the typescript side and say that "the version installed with the CLI is the version used", so the peer dependency is just "*" to let the winglang's version take over.